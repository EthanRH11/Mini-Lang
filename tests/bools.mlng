begin:

    bool newStatement = false;
    bool oldStatement = true;

    if(oldStatement){
        out_to_console("Hey, the true bool, is working in the if statement"); ...
    } else {
        out_to_console("We should not be here"); ...
    }

    if(newStatement){
        out_to_console("We should not be here") ...
    } else {
        out_to_console("Hey, the false bool, is working in the if statement"); ...
    }
end