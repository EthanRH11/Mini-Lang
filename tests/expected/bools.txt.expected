Hey, the true bool, is working in the if statement
Hey, the false bool, is working in the if statement